SYS_ATL/LoopIR_unification.py

```diff
@@ -701,10 +701,13 @@ def get_arg(fa):
                                      stmt_block[0].srcinfo)
             elif fa.type == T.bool:
                 if fa.name in self.bool_holes:
-                    return self.bool_holes[fa.name]
```

Comment:
@gilbo Can you take a look at this line? If the `self.bool_holes[fa.name]` was unchanged (False), that means it was an unused argument. Return `LoopIR.expr` in that case.